tests/pyccel/scripts/classes/classes_4.py

```diff
@@ -0,0 +1,17 @@
+# pylint: disable=missing-class-docstring,  missing-function-docstring, missing-module-docstring
+
+import numpy as np
+
+y = np.ones(4)
+
+class A:
+    def __init__(self : 'A', n : int):
+        self.x = np.ones(n)
+
+    def __del__(self : 'A'):
+        del self.x
```

Comment:
This `__del__` method is essentially identical to the default one. Maybe we can have it do something special, like printing a message?

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1508/commits/f414979b3676813938fa7478ad469f071b8c8942.